service/storage_service.cc

```diff
@@ -391,11 +391,15 @@ future<> storage_service::topology_state_load() {
         co_await tmptr->clear_gently(); // drop previous state
 
         tmptr->set_version(_topology_state_machine._topology.version);
+        tmptr->get_new()->set_version(_topology_state_machine._topology.version);
 
         auto update_topology = [&] (locator::host_id id, inet_address ip, const replica_state& rs) {
             tmptr->update_topology(ip, locator::endpoint_dc_rack{rs.datacenter, rs.rack},
                                    to_topology_node_state(rs.state), rs.shard_count);
+            tmptr->get_new()->update_topology(id, locator::endpoint_dc_rack{rs.datacenter, rs.rack},
```

Comment:
nit: indentation style in new code should be consistent at least with the pre-existing code around it.